packages/features/ee/workflows/lib/reminders/smsReminderManager.ts

```diff
@@ -60,6 +62,19 @@ export const scheduleSMSReminder = async (
 
   const senderID = getSenderId(reminderPhone, sender);
 
+  const verifiedNumbers = await prisma.verifiedNumber.findMany({
+    where: {
+      userId,
+    },
+  });
+
+  //SMS_ATTENDEE action does not need to be verified
+  //isVerificationPending is from all already existing workflows (once they edit their workflow, they will also have to verify the number)
+  const isNumberVerified =
```

Comment:
Here we make sure that the phone number we are trying to send an SMS to is verified (exceptions: see in the comment above)